page/jquery-ui/environments/bower.md

```diff
@@ -94,13 +94,13 @@ We have a few different options for using the files downloaded with Bower. The e
 	<h3>jQuery Mobile</h3>
 	<p>jQuery Mobile is a HTML5-based user interface system...</p>
 </div>
- 
+
 <script src="bower_components/jquery/dist/jquery.min.js"></script>
 <script src="bower_components/jquery-ui/jquery-ui.min.js"></script>
 <script>
 	$( "#projects" ).accordion();
 </script>
- 
+
```

Comment:
I apologize, but my editor ate this whitespace on file save.  Please let me know if I should put it back in.